nginx-config/default.conf

```
@@ -7,7 +7,10 @@ server {
     ssl_certificate_key /etc/letsencrypt/live/brokencrystals.com/privkey.pem;
 
     root         /var/www/html;
-
+    # Add Gzip support
+    gzip on;
```

Disable the compression for small files 

```suggestion
    gzip on;
    gzip_min_length 256;
```